app/graphql/types/weather_type.rb

```diff
@@ -33,7 +33,7 @@ def initialize(value, context, **kwargs, &block)
 
   field "id", Integer
   field "main", String
-  field "description", String, resolve: ->(obj, _args, _ctx) { obj["description"].to_s.humanize }
+  field "description", String, resolve: ->(obj, _args, _ctx) { obj["description"].to_s.humanize}
```

Comment:
Just curious what this line does (I guess I'm not very familiar with the syntax)

Reply:
https://apidock.com/rails/v5.2.3/ActiveSupport/Inflector/humanize